Xamarin.MacDev/MacOSXSdk.cs

```diff
@@ -39,8 +39,10 @@ public class MacOSXSdk
 			MacOSXSdkVersion.V10_10,
 			MacOSXSdkVersion.V10_11,
 			MacOSXSdkVersion.V10_12,
-			MacOSXSdkVersion.V10_13
-		};
+			MacOSXSdkVersion.V10_13,
+            MacOSXSdkVersion.V10_14,
+            MacOSXSdkVersion.V10_15
```

Comment:
I'm not 100% sure, but I don't think we should add 10.15 as a known version just yet. This list should contain macOS versions up to the previous version to the stable one (up to 10.14 in this case), and there are 2 reasons for that:
1. This list is used to populate the Minimum Target Version on VSM, if we always offer 10.15 users running on 10.14 could end up in an invalid situation (the Minimum target being 10.15 and trying to run that app on 10.14).
2. To handle that scenario VSM is already adding to the Minium Target Versions list the macOS version in which it is being run (if it does not already exist). 

I know all this sounds a bit tricky, and there are better solutions, but I think that's how it works now.

Besides removing it from this list, I'd leave the `V10_15` and `Catalina` new properties.